youslist.txt

```diff
@@ -6151,14 +6151,17 @@ odysee.com##div.ads__claim-item
 ohli24.net##div#exoall
 ohli24.net##div#exon
 ohli24.net##div.insty
+ohmynews.com###div10Sticky
```

Comment:
I think this one can be annoying, but it's for sponsoring their own site. I'll let them be.